packages/blade/src/components/Box/Box.tsx

```diff
@@ -136,6 +136,37 @@ const makeBoxProps = (props: BoxProps): KeysRequired<Omit<BoxProps, 'testID' | '
   };
 };
 
+const _Box: React.ForwardRefRenderFunction<BoxRefType, BoxProps> = (props, ref) => {
```

Comment:
diff is a bit messed up here but I essentially moved the jsdoc from `_Box` to `Box` (unrelated to this issue)